app/controllers/products_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ProductsController < ApplicationController
+
+	def index
+		@products = Product.order(id: :asc)
+		render :index
+	end
+
+	def show
+    @vendor = Vendor.find(params[:id])
+    render :detail_view
+  end
```

Comment:
Some of the indents in this file as well are a off. It's a little thing, but in a large scale apps it really helps with readability (and in some other languages if the whitespace is off the program won't work as expected) so it's a good habit to practice now.